doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -546,6 +546,11 @@ The following list summarizes the most important changes inherited from the upst
   * Support for Administrator Commissioning Cluster, which allows enabling or disabling the commissioning window on a Matter device.
     This is required by the Matter multi-admin functionality.
 
+Memfault
```

Comment:
I believe we don't need a mention about the Memfault FW SDK version update in our changelog, as it is an external product and we don't mention its version in any of the NCS docs. I will confirm this with my team and come back to this asap.

Reply:
Discussed offline, and agreed to remove the changelog entry. PR has been updated.